### ISSUE TITLE
Scrape Contacts

### ISSUE THREAD

**Issue by firmai** [linkedtales/scrapedin-linkedin-crawler#33]
Nice project, will this also scrape emails?

**Comment by leonardiwagner** [comment]
yes, it is, please collaborate sending the code to integrate with this project, thank you.

**Comment by jeremiah-94** [comment]
I have looked through the project but find that emails cannot be scrapped if they are not public. That results in a lower % of emails that are available and obtainable via scraping. I ended up resorting to a paid solution Mantheos[https://profiles.mantheos.com/](url) that has an API that allows me to extract contact information at mass whether it is public or private profiles.
